patches/chrome-browser-ui-views-tabs-tab.cc.patch

```diff
@@ -2,15 +2,23 @@ diff --git a/chrome/browser/ui/views/tabs/tab.cc b/chrome/browser/ui/views/tabs/
 index 444832f1c6e5fb98cef09887796e8d4cdaba6550..bf3d597c4cb0e2151ed33023d26b46b15f4a3430 100644
 --- a/chrome/browser/ui/views/tabs/tab.cc
 +++ b/chrome/browser/ui/views/tabs/tab.cc
-@@ -420,6 +420,7 @@ void Tab::Layout(PassKey) {
+@@ -324,6 +324,7 @@ void Tab::Layout(PassKey) {
+     // Height should go to the bottom of the tab for the crashed tab animation
+     // to pop out of the bottom.
+     favicon_bounds.set_y(contents_rect.y() +
```

Comment:
`#define set_y(POINT) set_y(contents_rect.y() + Center(contents_rect.height(), gfx::kFaviconSize))`

Reply:
actually that is maybe a bit risky so I guess this patch is ok